help/Makefile.am

```diff
@@ -1,7 +1,7 @@
 # Generated automatically from Makefile.in by configure.
```

Comment:
Is this true? If so why are we modifying it? If not can we delete it?

Reply:
I think it's there so it shows up in the generated Makefile. The next line is definitely wrong, though.